src/OSPSuite.InstallationValidator.Core/Presentation/MainPresenter.cs

```diff
@@ -60,14 +64,30 @@ public void Abort()
          }
          catch (OperationCanceledException)
          {
-            _dialogCreator.MessageBoxInfo(Constants.Captions.TheValidationWasCanceled);
+            logText(Constants.Captions.TheValidationWasCanceled);
+         }
+         catch (Exception e)
+         {
+            logException(e);
          }
          finally
          {
             View.ValidationIsRunning(false);
          }
       }
 
+      private void logText(string theTextToLog)
+      {
+         View.AppendText(theTextToLog);
+      }
+
+      private void logException(Exception e)
+      {
+         View.AppendText(Constants.Captions.Exceptions.ExceptionSupportMessage(_configuration.IssueTrackerUrl));
```

Comment:
there is this logText method that is defined but not used at all. I think that should be updated but can be done as a refactgoring in a future PR